cmd/thanos/compact.go

```diff
@@ -212,7 +217,7 @@ func runCompact(
 
 	ctx, cancel := context.WithCancel(context.Background())
 	f := func() error {
-		if err := compactor.Compact(ctx); err != nil {
+		if err := compactor.Compact(ctx, concurrency); err != nil {
```

Comment:
Could we move this to the constructor instead?

`NewBucketCompactor(..., concurrency)`

Reply:
Agee